src/components/blocksmith-field/components/instagram-media-block-modal/InstagramMediaPreview.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useRef } from 'react';
+
+const loadInstagramSDK = (embedParentRef) => {
+  const script = document.createElement('script');
+  script.async = true;
+  script.src = 'https://www.instagram.com/embed.js';
+  script.onload = () => {
+    window.instgrm.Embeds.process();
+  };
+  embedParentRef.current.appendChild(script);
+}
+
+export default function InstagramMediaBlockPreview({ formState }) {
+  
+  const { hidecaption, id } = formState.values;
+  const embedParentRef = useRef();
+
+  useEffect(() => {
+    loadInstagramSDK(embedParentRef);
+    embed();
+  }, [id, hidecaption]);
+
+  const clean = () => {
+    Array.from(embedParentRef.current.children).forEach((child) => {
+      embedParentRef.current.removeChild(child);
+    });
+  }
+
+  const embed = () => {
+    clean();
```

Comment:
will this cause a problem if clean comes after loadInstagramSDK? For imgur the script is loaded after the clean